paper-api/src/main/java/io/papermc/paper/datacomponent/item/CustomModelData.java

```diff
@@ -82,6 +83,16 @@ interface Builder extends DataComponentBuilder<CustomModelData> {
         @Contract(value = "_ -> this", mutates = "this")
         CustomModelData.Builder addFloats(List<Float> floats);
 
+        /**
+         * Sets multiple floats to this custom model data.
+         *
+         * @param floats the floats
+         * @return the builder for chaining
+         * @see #floats()
+         */
+        @Contract(value = "_ -> this", mutates = "this")
+        CustomModelData.Builder setFloats(List<Float> floats);
```

Comment:
this can be just floats. based in the format of the "set" methods here.
Same with others sets methods